hs_core/models.py

```diff
@@ -718,26 +729,25 @@ def update(cls, element_id, **kwargs):
         metadata_obj = kwargs['content_object']
         # get matching resource
         resource = BaseResource.objects.filter(object_id=metadata_obj.id).first()
-
+        publisher_CUAHSI = "Consortium of Universities for the Advancement of Hydrologic Science, Inc. (CUAHSI)"
```

Comment:
@pkdash @horsburgh Just confirm: even if we decide to allow resources without content files to be published, Publisher element should not be allowed to be updated, right? We may set up a different name for Publisher, but once it is set, it is final and cannot be updated, right?